### DISCUSSION TITLE
Is there a way to get the precipitation chances within the hourly forecast endpoint?

### DISCUSSION THREAD

**Discussion by simurgh9** [weather-gov/api#299] [Q&A] [answered]
When you hit `https://api.weather.gov/gridpoints/{wfo}/{x},{y}/forecast/hourly`, an information block for a period looks like this:

```json
{
    "number": 58,
    "name": "",
    "startTime": "2020-08-22T06:00:00-05:00",
    "endTime": "2020-08-22T07:00:00-05:00",
    "isDaytime": true,
    "temperature": 19,
    "temperatureUnit": "C",
    "temperatureTrend": null,
    "windSpeed": "11 km/h",
    "windDirection": "SSE",
    "icon": "https://api.weather.gov/icons/land/day/tsra_hi?size=small",
    "shortForecast": "Slight Chance Showers And Thunderstorms",
    "detailedForecast": ""
}
```
Is there a way to get the precipitation chances within this block or maybe calculate them for a block using the information given within? For the endpoint `https://api.weather.gov/gridpoints/{wfo}/{x},{y}/forecast`, I see that when there is a precipitation chance, it is usually mentioned in a sentence within the `detailedForecast` variable but unfortunately this variable is always empty for the hourly forecast.

**Comment by nearwood** [comment]
It's available with the raw data call: https://www.weather.gov/documentation/services-web-api#/default/get_gridpoints__wfo___x___y_

```json
"probabilityOfPrecipitation": {
      "uom": "wmoUnit:percent",
      "values": [
        {
          "validTime": "2020-09-20T14:00:00+00:00/PT4H",
          "value": 0
        },
        {
          "validTime": "2020-09-20T18:00:00+00:00/PT6H",
          "value": 1
        },
        {
          "validTime": "2020-09-21T00:00:00+00:00/PT12H",
          "value": 0
        },
```

**Comment by grantiago** [comment]
<pre>https://api.weather.gov/gridpoints/{wfo}/{x},{y}</pre>
is correct.

**Comment by simurgh9** [comment, marked as answer]
Came here after years, seems that the hourly forecast endpoint now includes `probabilityOfPrecipitation`. :)
